diagrams/sequencediagram.txt

```diff
@@ -62,49 +60,32 @@ m_farcasterd<-m_syncer:Ctl Hello [from Syncer Bitcoin (if newly launched) OR Swa
 group for Bob swap role
 t_syncer <- t_swap : Watch Arbitrating Funding Address
 end
-t_swap -> t_wallet : Msg MakerCommit
-t_wallet -> t_swap : Msg Reveal (taker is sender)
 t_swap -> peerd : Msg Reveal (taker is sender)
 peerd -> m_swap : Msg Reveal (taker is sender)
-group for Alice swap role
-m_swap -> m_wallet : Msg Reveal (taker is sender)
-end
 group for Bob swap role
-m_swap -> m_swap : ADD PENDING Msg Reveal
+m_swap -> m_swap : ADD Msg CoreArb
 m_swap -> m_farcasterd : Ctl FundingInfo
 m_swap -> m_syncer: Watch Arbitrating Funding Address
-m_swap <- m_syncer: Arbitrating Funding event
-m_farcasterd<-m_swap: Ctl FundingCompleted Bitcoin
-m_swap->m_wallet: Ctl Tx::Funding
-m_swap<-m_wallet: Ctl FundingUpdated
-m_swap -> m_wallet : SEND PENDING Msg Reveal (taker is sender)
+
 end
-m_wallet -> m_swap : Msg Reveal (maker is sender)
 peerd <- m_swap : Msg Reveal (maker is sender)
 peerd -> t_swap : Msg Reveal (maker is sender)
-group for Alice swap role
-t_swap -> t_wallet : Msg Reveal (maker is sender)
-end
+
 group for Bob swap role
-t_swap -> t_swap : ADD PENDING Msg Reveal
+t_swap -> t_swap : ADD Msg CoreArb
```

Comment:
I'm not sure where you are seeing CoreArb as pending? But CoreArb is still wrong, it adds the Reveal message to the state.

Reply:
2d0938a51c50f50dbef8b49e40de7b9078ab9f54

Reply:
ah yeah - I assumed that because there was no CoreArb sent by the counterparty, this was the taker already creating CoreArb to send later (pending), but that was mistaken and it now makes perfect sense again, thanks!